app/lib/appframe.dart

```diff
@@ -27,15 +28,20 @@ class _AppFrameState extends State<AppFrame> {
         selectedIndex: currentPageIndex,
         destinations: <Widget>[
           NavigationDestination(
-            selectedIcon: const Icon(Icons.home),
-            icon: const Icon(Icons.home_outlined),
+            selectedIcon: const Icon(Icons.home_outlined),
+            icon: const Icon(Icons.home),
             label: AppLocalizations.of(context)!.frameHome,
           ),
           NavigationDestination(
-              selectedIcon: const Icon(Icons.list_outlined),
-              icon: const Icon(Icons.list),
+              selectedIcon: const Icon(Icons.person_outlined),
+              icon: const Icon(Icons.person),
               label: AppLocalizations.of(context)!.framePatients
           ),
+          NavigationDestination(
+              selectedIcon: const Icon(Icons.map_outlined),
```

Comment:
I planned to use the map for my map screen, are there good alternatives?

Reply:
changed this introducing a new constant package. It would be more efficient to keep all these constants in a single file, so we only need to update one place when changes are required I guess